packages/deployer/src/server/handlers/voice.ts

```diff
@@ -37,14 +37,15 @@ export async function speakHandler(c: Context) {
     const mastra: Mastra = c.get('mastra');
     const agentId = c.req.param('agentId');
     const { input, options } = await c.req.json();
+    const { speakerId, ...providerOptions } = options ?? {};
 
     const audioStream = await getOriginalSpeakHandler({
       mastra,
       agentId,
-      body: { text: input, speakerId: options?.speakerId },
+      body: { text: input, speakerId, ...providerOptions },
```

Comment:
Should we add providerOptions as a key instead? This would prevent destructuring and allow us to update the args in the future without breaking user flows.

```suggestion
      body: { text: input, speakerId, providerOptions },
```